.github/ISSUE_TEMPLATE/update-an-analysis-template.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Update an analysis
+about: Use this issue template to describe an update to a pre-existing analysis module
+title: 'Updated analysis:'
+labels: updated analysis
+assignees: ''
+
+---
+
+<!--Hi there! Please take a moment to fill out the template below.-->
+
+<!-- Check all those that apply or remove this section if it is not applicable.-->
+
+## What analysis module should be updated and why?
+
+
+
+
+## What changes need to be made? Please provide enough detail for another participant to make the update.
+
+
+
+
+## What input data should be used? Which project/data were used in the version being updated?
```

Comment:
Again, maybe separate these questions, so it's only one question per each different line